requirements.txt

```diff
@@ -1,30 +1,3 @@
-Jinja2>=2.8
-Logbook>=0.7.0
-MarkupSafe>=0.23
-PyYAML>=3.11
-Zendesk>=1.1.1
-backports.ssl-match-hostname>=3.4.0.2
-certifi>=2019.6.16
-chardet>=3.0.4
-click>=6.2
-ecdsa>=0.11
-genologics>=0.4.6
-httplib2>=0.9
-idna>=2.8
-ipdb>=0.8
-ipython>=3.0.0
-matplotlib>=1.4.0
-numpy>=1.9.0
-oauth2>=1.5.211
-paramiko>=1.14.1
-py-trello>=0.1.6
-pycryptodome>=3.8.2
-pyparsing>=2.0.1
-python-dateutil>=1.5
-pytz>=2014.7
-requests>=2.22.0
-simplejson>=3.6.4
-six>=1.8.0
-tornado>=4.0.2
-urllib3>=1.25.3
-wsgiref>=0.1.2
```

Comment:
What were we thinking? I also spot some more cruft from ancient times that has been removed :+1: